docs/guides.md

```diff
@@ -1426,3 +1426,48 @@ secureboot-certs install default KEK.auth default latest
 ```
 
 This may be done with any PK, KEK, db, or dbx.
+
+## Use a VNC client like vncviewer
```

Comment:
> Is this specific to vncviewer? How about just "Use a VNC client" and mention vncviewer only in the text?

Yes!

> Lastly, maybe this would be a better place: https://xcp-ng.org/docs/guests.html#vm

I thought of putting it there originally, but it's a guide that I wrote right? :p

Reply:
The guides page is not for all guides, it's for guides that don't fit elsewhere :)